cirq-google/cirq_google/devices/known_devices.py

```diff
@@ -58,11 +60,9 @@ def create_device_proto_from_diagram(
     durations_picos: Optional[Dict[str, int]] = None,
     out: Optional[device_pb2.DeviceSpecification] = None,
 ) -> device_pb2.DeviceSpecification:
-    """Parse ASCIIart device layout into DeviceSpecification proto.
-
+    """[Deprecated] Parse ASCIIart device layout into DeviceSpecification proto.
```

Comment:
I don't understand why we don't have a @_compat.deprecated decorator on this function.  It should just be one line, so I don't think it will be burdensome at all, and it is much more obvious than a docstring.

Reply:
Sounds good. Tests need to include `assert_deprecation()` contexts but there aren't many of them. Will update

`_SYCAMORE_DURATIONS` would be trickier to deprecate, though, since it's a constant. Might need something similar to examples in [PEP 562](https://peps.python.org/pep-0562/) and probably not worth the effort.

Reply:
That's fine.  My comment was mainly about the function.  _SYCAMORE_DURATIONS is a constant and it is already designated as private from the underscore, so I do not think we need a deprecation warning for it.